ts/theme/variables.ts

```diff
@@ -67,9 +67,26 @@ const customVariables = Object.assign(materialVariables, {
 
   // Icon
   iconFamily: "Entypo",
-  iconSmall: 16,
-  iconBase: 20,
-  iconLarge: 46, // WIP should these also depend on fontSizeBase? (also, iconSizeLarge is already taken)
+  iconSizeBase: 20,
```

Comment:
@fgiobergia @cloudify do you think it is more appropriate to set this to the real base size of the icons: 24px?

Reply:
what do you mean by "real"?

Reply:
@matteodesanti in the last call said he designs the icons (or most of them) as 24x24 pixel.

Reply:
but these are vector icons, aren't they?

Reply:
Yes it is not a scaling problem, it like saying "hey default/basic size of the icons in the app is 24", so we can set this size in the theme and if you don't specify a size for an Icon component it will be 24.

Reply:
ok for me - in general I think size in pixel doesn't matter much as mobile devices will have different DPIs

Reply:
Considering that [all dimensions in React Native are unitless, and represent density-independent pixels](All dimensions in React Native are unitless, and represent density-independent pixels), it seems like a bit of a stretch to me to use 24 as the base size because @matteodesanti previews them as 24x24 pixels images. 

I would probably try to find out whether any kind of standard exists for defining sizes relative to another (I'm thinking something like the `h1` ... `h6` tags -- something like [this](https://stackoverflow.com/a/5410145/1801231) )

Reply:
ok I suggest to go ahead and merge this PR while opening a story (in the icebox) for reviewing this topic at a later time (link this thread from the story) - @fgiobergia can you pls do that? thanks

Reply:
By the way, if we wanted to adopt the new coefficients `[2,1.5,1.3,1,0.8,0.7]` (let's call them `Fs` for "Future sizes" with a new base font size `x` (to be determined), and starting from a font size `Cf` (Current fontsize) = 20 and the coefficients I already used `Cs = [2.3, 1.7, 1.3, 1, 0.8, 0.6]` while minimizing the impact on the parts of the app that already use these sizes (measured in terms of sum of square differences between the current and the future font sizes), we should adopt a new base font size `x` ~ 21.65 (or 22 if you like round numbers). The strong assumption here is that I got the math right ([math attached](https://user-images.githubusercontent.com/107715/40837752-ccf2d1c4-659b-11e8-9c7f-7766cf87b80a.png)). I guess I'll go back to coding now